src/components/Navigation/Navigation.tsx

```diff
@@ -1,41 +1,31 @@
-import React, { FC } from 'react';
-import { makeStyles } from '@material-ui/core/styles';
-import clsx from 'clsx';
+import React, { FC, useCallback } from 'react';
+import { useHistory } from 'react-router-dom';
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 import { faChevronLeft } from '@fortawesome/free-solid-svg-icons';
+import { IconButton } from '@material-ui/core';
 
-import styles from './Navigation.module.scss';
-
-const useStyles = makeStyles(
-  theme => ({
-    navigation: {
-      backgroundColor: theme.palette.background.default,
-      color: theme.palette.text.primary
-    }
-  }),
-  { name: 'Navigation' }
-);
+import { Pathway } from 'pathways-model';
+import useStyles from './styles';
 
 interface Props {
-  name?: string;
-  onBack?: () => void;
+  pathway?: Pathway | null;
```

Comment:
Since this was moved to be part of the Builder View only this should be required (assuming you agree with my previous comment about pathway being required for the Builder view)